view-samples/shopping-cart/README.md

```diff
@@ -0,0 +1,56 @@
+# Shopping Cart
+
+## Summary
+
+The JSON template is designed for an online shopping cart. It displays items within the cart, including product details, pricing, shipping information, and user interactions like adding or removing items.
+
+Clicking on 'Add to Cart' updates the 'AddedToCart' field to true and changes the display text to 'Remove from Cart'
+
+![screenshot of the sample](./assets/screenshot.png)
+
+## View requirements
+
+Column Name               | Type
+--------------------------|----------------------------------------
+Title                     | Single line of text
+Description               | Multiple lines of text
+Price                     | Currency
+OldPrice                  | Currency
+ImageUrl                  | Single line of text
+Offer                     | Single line of text
+CalculatedPrice           | Calculated (calculation based on other columns)(Formula = OldPrice-Price)
```

Comment:
Please write the type of Calculated column.

![image](https://github.com/user-attachments/assets/c812c6fb-8060-44f7-b1ec-94463f6aa228)

Reply:
I have updated the formula and the type.